ansible/environments.yml

```diff
@@ -103,3 +103,18 @@
           image_name: "d324da4b-b827-44fb-8331-bfad41a28aca"
       tags:
         - dev2-env
+
+    - name: Ensure ae-uat-uswest2 environment stack
+      cloudformation:
+        stack_name: "uat"
```

Comment:
Very similar to ae-uat above, may cause mistakes by typo. Close name on purpose?

Reply:
bad/wrong naming indeed, but let it go